src/mon/OSDMonitor.cc

```diff
@@ -5072,6 +5072,20 @@ void OSDMonitor::get_pools_health(
           detail->push_back(s);
       }
     }
+
+    if (sum.num_objects > 0 && pool.application_metadata.empty()) {
```

Comment:
Yeah, nowhere yet.  The feature sticky feature bit update is buried in MonmapMonitor somewhere but so far there has been no addition logic as part of that transition.